### ISSUE TITLE
ECJ Version and no Output in console

### ISSUE THREAD

**Issue by dovogt** [TwoStone/gradle-eclipse-compiler-plugin#5]
I have a big project and the compiler messages are written to a file. Nevertheless, you are written in the console. Is it possible to suppress this?

	
```
options.fork = false
options.encoding = "cp1252"
options.warnings = false
options.compilerArgs << '-properties' << '.settings/org.eclipse.jdt.core.prefs' << '-log' << buildDir.getAbsolutePath() + 'reports/compiler.log' << '-proc:none'

```

Can I specify the ECJ version to use?

**Comment by zsalab** [comment]
define the version is available with #4 PR (unfortunately not merged yet)
you will see example there as well

unfortunately I do not know the answer to your other question

**Comment by dovogt** [comment]
Thanks, it worked.

I keep looking for a solution. If I found one. I write it in the post
